src/peer.js

```diff
@@ -147,7 +147,7 @@ class Peer extends EventEmitter {
 
   ping (cb) {
     var start = Date.now()
-    var nonce = crypto.pseudoRandomBytes(8)
+    var nonce = crypto.randomBytes(8)
```

Comment:
This shouldn't be necessary since browserify has the `pseudoRandomBytes` method. I'm not sure why your build doesn't have it, maybe update browserify?